### PR TITLE
Add Bandit

### PR DESCRIPTION
**Pull request by bdwyer2**
Use bandit to check code security

### CODE REVIEW

**Comment by xuhdev** [comment]
The use of bandit LGTM. Are we going to do this for all models?

**Comment by bdwyer2** [comment]
> The use of bandit LGTM. Are we going to do this for all models?

This is just a proof of concept for now, we can discuss at the next scrum

**Comment by gdequeiroz** [comment]
LGTM. I didn't know about this tool. I did a quick reading and it looks good.

File: .bandit [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[bandit]
+exclude: /tests,/training
```
Comment (xuhdev) [inline .bandit:2]: Any reason to exclude `/training`?

Comment (bdwyer2) [reply .bandit:2]: There are many security issues with training, fixing it is going to be a project. We should discuss if we need to do this and/or want to do this.

Comment (xuhdev) [reply .bandit:2]: OK then. Shall we hold this PR for a couple of days before we make the decision?

Comment (MLnick) [reply .bandit:2]: Training targets WML (or local) and happens in a sandboxed environment - i.e. doesn't expose any public endpoints etc. 

I think we can exclude the `/training` subfolder

**Pull request merged**